### ISSUE TITLE
Can catch pokemon but Pokestops are "out of range"

### ISSUE THREAD

**Issue by salamiritter** [finndev/PokeBuddy#958]
Hey guys,

the bot stopped farming pokestops just today. 
It still catches Pokemon and it still displays the name of the pokestop but also:
[Farm Pokestops] OutOfRange -> Received XP: 0
Anyone having the same problem?

**Comment by Kokainium** [comment]
same for me :S

**Comment by salamiritter** [comment]
![pokebuddy - a pokemon go bot](https://cloud.githubusercontent.com/assets/20991860/17624300/4f931660-60a4-11e6-99c5-5927c31be817.png)

**Comment by Kokainium** [comment]
yes

**Comment by futi666** [comment]
![dzt](https://cloud.githubusercontent.com/assets/20840876/17624715/1bf48a62-60a6-11e6-8127-99237644f7db.png)

**Comment by Drorien** [comment]
Same problem, guys do you know how to fix that?

**Comment by raphaelmaron** [comment]
same here, and when I try on cellphone or nox it keeps saying "try again later"

some kind of ban, maybe?

**Comment by AlexCtasy** [comment]
Yeah, same for me here, it's a softban.. But I tried to unban with quickspin btw still softbanned, so I guess just need wait some hours to get unbanned now :S

**Comment by raphaelmaron** [comment]
for me its almost 12 hours since it happened

**Comment by anonimo345423** [comment]
happening here too, same also if i take a pokestop by hand

**Comment by DUSKvsDAWN** [comment]
Same problem here, don't know what cause it. Doesnt see like ban since we can catch Pokemon........ except that I have no pokeballs left now.

**Comment by gattu95** [comment]
hello too that mistake , I can capture Pokemon and win in gyms but pokestop not work , now after 30 minutes everything is back to normal , the error I came out after 5 hours of pokebuddy start , I think is a mini ban

**Comment by gattu95** [comment]
Now after 10 minutes the same problem, someone has solved ?

[P - 19:29:25] [Catch Pokemon] Found: 1 catchable Pokemon
[P - 19:29:28] [Catch Pokemon] We caught a Meowth (Probability: 47.34%) with CP 184, used 1 x ItemPokeBall and received 160 XP
[P - 19:29:29] [Farm Pokestops] Spinning Pokestop: 'Broadway Theater'
[P - 19:29:29] [Farm Pokestops] OutOfRange -> Received XP: 0

**Comment by andreanac** [comment]
Its a temporary Ban, max 12hr.

**Comment by gattu95** [comment]
but what caused it ? I often and I have to reboot the router to get it going

**Comment by andreanac** [comment]
for me is the same when i reboot the router

**Comment by gattu95** [comment]
Italiano? siccome Hanno Bannato Il mio amico non lo usero piu , usava anche lui bot , Mi sono spaventato , comunque ti ringrazio 

O inglese? 
as They Banned My friend do not I will use more , also used him bot , I was scared , anyway than

**Comment by gattu95** [comment]
Anyway thank you
